frontend/src/routes/(root)/(logged)/run/[...run]/+page.svelte

```diff
@@ -731,6 +735,7 @@
 						flowDone={job.type == 'CompletedJob'}
 					/>
 				{/if}
+				<JobProgressBar {job} {scriptProgress} class="py-4" hideStepTitle={true}/>
```

Comment:
should surround by {#if scriptProgress}, lots executed for empty scriptProgress otherwise (even if you did put the if on the dom related code, spawning a svelte component is not free)